.github/workflows/ci.yml

```diff
@@ -172,7 +181,7 @@ jobs:
       - frontend-unit-tests
       - frontend-e2e-tests
       - build-skip-check
-    if: ${{ needs.build-skip-check.outputs.skip }} == false
+    if: needs.build-skip-check.outputs.skip == 'false'
```

Comment:
Turns out we don't need the `{{ }}` jinja braces, and this is now actually filtering whereas it was not before.